### PR TITLE
Block sign with timestamp and previous block hash as payload.

### PR DESCRIPTION
**Pull request by humb1t**

### CODE REVIEW

File: iroha/src/sumeragi.rs [hunk 1]
```diff
@@ -225,11 +232,19 @@ impl Sumeragi {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use std::convert::TryInto;
+    use ursa::signatures::{ed25519::Ed25519Sha512, SignatureScheme};
 
     #[test]
     #[should_panic]
```
Comment (takemiyamakoto) [inline iroha/src/sumeragi.rs:239]: Generally even running on 1 node (i.e., with no peers) should be fine. We just assume the user knows what they are doing and don't need the coolness of blockchain redundancy.

Comment (humb1t) [reply iroha/src/sumeragi.rs:239]: I think it's more about @eadventurous changes. @eadventurous please create follow-up task

File: iroha/src/sumeragi.rs [hunk 2]
```diff
@@ -149,16 +155,17 @@ impl Sumeragi {
             self.proxy_tail().clone(),
         )
         .await;
-        Ok(())
+        Ok(block)
     }
 
     pub async fn handle_message(&mut self, message: Message) -> Result<(), String> {
```
Comment (takemiyamakoto) [inline iroha/src/sumeragi.rs:161]: Do we assume that the block has been fully validated before we handle this?

Comment (eadventurous) [reply iroha/src/sumeragi.rs:161]: I don't see why we should return the block. From what I understand, when the block is created we only send the `Created` message and then wait for `Commited` message to actually commit the block to Kura and WSV.

File: iroha/src/sumeragi.rs (hunk 1 above)
Comment (eadventurous) [reply iroha/src/sumeragi.rs:239]: Added task in JIRA https://soramitsu.atlassian.net/browse/RH2-72?atlOrigin=eyJpIjoiZGJkYzM1MWE5ZDRiNGQ5YjlmYTE0ZDY4MGVhYzI4NDUiLCJwIjoiaiJ9

File: iroha/src/sumeragi.rs (hunk 2 above)
Comment (humb1t) [reply iroha/src/sumeragi.rs:161]: Right now - we do not use this method at all. So I think we will make changes after adoption

**Pull request merged**